vendor/theme/templates/layouts/post-donate-page-with-recurring-ask.liquid

```diff
@@ -68,8 +68,31 @@
 				<p>
 					<b>Werden Sie Teil unserer Gemeinschaft. 17,462 Fördermitglieder unterstützen SumOfUs regelmäßig:</b>
 				</p>
-			</div>	
-			<div class="fundraiser-bar__step-panel script-dependent">
+			</div>
+      <div class="es-copy">
+        <p>
+          <b><span class="thank-you-firstname">Gracias </span>por ayudar a construir un mundo en el que las personas pasan antes que el lucro.</b>
```

Comment:
Hi fam! Probably signed off already but have you considered: _"un mundo en el que las personas **vienen** antes que el lucro"_?

Reply:
Vince! Fancy seeing you here! :D I haven't considered it, as the extent of my Spanish is "Gracias", "yo soy un camaron" and "mi gato esta loco" :| But I will implement your suggestion!